### PR TITLE
Mixedcontents

### PR DESCRIPTION
**Pull request by Carreau**
@KesterTong Yes sure we can merge mixed contents. I tend to forgot we have two branches. 

I'll just let this PR open a few hours time to review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Google Drive support for Jupyter Notebook
 
-This repository contains a custom
+This repository contains customs
```
Comment (KesterTong) [inline README.md]: customs => custom

Comment (rgbkrk) [inline README.md]: Errr, looks like "a custom" is right.

"...contains a custom Contents class that..."

**Comment by KesterTong** [comment]
ok looks good to me.  Feel free to merge when you've reviewed yourself.

File: README.md (hunk 1 above)
Comment (Carreau) [inline README.md]: It contains 2 classes, Client Side and mixed content so should it be plurals without `a` ?

**Comment by Carreau** [comment]
Anyway typo can be corrected in subsequent PRs. 

Merging.

File: README.md (hunk 1 above)
Comment (rgbkrk) [inline README.md]: Ah yeah I read it as is without knowing context.

**Pull request merged**